DEVELOPER.md

```diff
@@ -114,6 +114,17 @@ networks:
     assignments:              # assign static addresses to devices
       magda-core: 1
 
+# not part of magda conf, only to illustrate tunnel usage
```

Comment:
please put the example in a seperate code block

Reply:
i put there because it's part of the networks.yml so how can i put it into an extra block?

Reply:
i mean should i put it into an extra section like: uplink via tunnel -> your networks.yml is gonna look like this?